articles/ssc17dot2.md

```diff
@@ -129,6 +129,13 @@ Parser plugins went through a major rewrite. They are now bundled with:
 * A view template to tell SSC how to show these fields.
 * The logic to map data from third-party file exports to these fields provided with bundles.
 
+#### Plugin Dependecies Note:
+If a plugin has any dependencies on javaEE packages, the plugin developer must bundle the necessary javaEE jars into the plugin's own library path, and must not rely on these packages being available from the JRE.  The JavaEE modules have been removed from current versions of Java post-Java8 in SSC 18.20.  
+ 
```

Comment:
replace "in SSC 18.20" with "and will not be available in SSC versions after 18.20."